pkg/test/localauth/localauth.go

```diff
@@ -0,0 +1,21 @@
+package localauth
+
+import (
+	"context"
+	"errors"
+)
+
+const testPassword = "test"
+
+type TestLocalAuthenticator struct{}
+
+func (a *TestLocalAuthenticator) GenerateAdminPassword(_ context.Context) (string, error) {
+	return testPassword, nil
+}
+
+func (a *TestLocalAuthenticator) ComparePassword(_ context.Context, pw []byte) error {
+	if string(pw) != testPassword {
+		return errors.New("mismatched password")
```

Comment:
Wouldn't this need to be bcrypt.ErrMismatchedHashAndPassword? Alternatively, we could have our own error for this.